cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.md

```diff
@@ -593,6 +593,49 @@ The following table gives an overview of various modules in Python 3 used for XM
 
 To protect your application from the applicable attacks, [two packages](https://docs.python.org/3/library/xml.html#the-defusedxml-and-defusedexpat-packages) exist to help you sanitize your input and protect your application against DDoS and remote attacks.
 
+## Semgrep Rules
+
+Semgrep is a command-line tool for offline static analysis. Use pre-built or custom rules to enforce code and security standards in your codebase.
```

Comment:
@salecharohit please add here a link to the Semgrep site or github

Reply:
6f08615d7ee156ab4300fbb1fa5fcd47c0a4a70a done